schema_based_on_diagram.sql

```diff
@@ -0,0 +1,63 @@
+/* Creating tables for clinic database and primery keys*/
+
+create table patients(
+    id int generated always as identity,
+    name varchar(60),
+    date_of_birth date,
+    primary key(id)
+)
+
+create table medical_histories(
+    id int generated always as identity,
+    admitted_at timestamp,
+    patient_id int,
+    status varchar(60),
+    primary key(id)
+);
+
+create table invoices(
+    id int generated always as identity,
+    total_amount decimal,
+    generated_at timestamp,
+    payed_at timestamp,
+    medical_history_id  int,
+    primary key(id)
+);
+
+create table invoices_items(
+    id int generated always as identity,
+    unit_price decimal,
+    quantity  int,
+    total_price  decimal,
+    invoice_id  int,
+    treatment_id int,
+    primary key(id)
+);
+
+create table treatments(
+    id int generated always as identity,
+    type varchar(60),
+    name varchar(60),
+    primary key(id)
+);
+
+/*Creating  foreign keys for the relationship between one to many tables */
+
+ALTER TABLE patients ADD FOREIGN KEY (id) REFERENCES medical_histories(id);
+ALTER TABLE invoices ADD FOREIGN KEY (id) REFERENCES invoices_items(id);
```

Comment:
- [ ] Kindly notice that you shouldn't create the column for the relation between the `patients` and `medical_histories` in the `patients` table, also you are naming the column `id` the same name as the primary key.
- [ ] Also you shouldn't create a column for the relation between the `invoices` and `invoices_items` in the `invoices` table, also you are naming the column `id` the same name as the primary key.
- [ ] So kindly remove these two lines.